hexchat/snapcraft.yaml

```diff
@@ -11,27 +11,26 @@ confinement: strict
 apps:
   hexchat:
     command: desktop-launch $SNAP/bin/hexchat
-    plugs: [network, network-bind, x11, unity7]
+    plugs:
+      - unity7
+      - gsettings
+      - pulseaudio
+      - network
+      - network-bind
+      - home
 parts:
   hexchat:
```

Comment:
Same, as part of our best practices (sorry, that should be documented), the order here should be:
```
  source:
  plugin:
  snap:
  build-packages:
  after:
```
We don't order them in ascii order, but in a logical one.